obal/data/ansible.cfg

```diff
@@ -2,7 +2,6 @@
 inventory = package_manifest.yaml
 retry_files_enabled = False
 transport = local
-callback_whitelist = profile_tasks
```

Comment:
Could you explain why this is removed?

Reply:
Ohh, I can put that back. It was mostly just muddling my output. I did wonder if anybody actively used this output. If not, its just eating screen space for users.

Reply:
Maybe make it at least a separate commit so it's clearly an unrelated change.